src/cdk/overlay/overlay-ref.ts

```diff
@@ -368,7 +368,7 @@ export class OverlayRef implements PortalOutlet, OverlayReference {
 
   /** Toggles the pointer events for the overlay pane element. */
   private _togglePointerEvents(enablePointer: boolean) {
```

Comment:
I was debating renaming this at the same time to `_disablePointerEvents` since it never really does enable them anymore, it just disables or defaults...

Reply:
IMO "toggle" still makes sense since the default is for them to be enabled.

Reply:
The default is in a css class though, this code really doesn't know and shouldn't care what that default is. Just my opinion though 🤷